server/pom.xml

```diff
@@ -62,6 +62,10 @@
             <artifactId>commons-lang</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-collections4</artifactId>
```

Comment:
This is bringing in org.apache.commons.collections4.iterators.ReverseListIterator, which is used in DruidCoordinatorHadoopSegmentMerger.